optional_plugins/html/bootstrap_library_config.json

```diff
@@ -379,53 +379,20 @@
     "@hr-border": "@gray-lighter"
   },
   "css": [
-    "print.less",
     "type.less",
-    "code.less",
     "grid.less",
     "tables.less",
     "forms.less",
-    "buttons.less",
-    "responsive-utilities.less",
-    "button-groups.less",
-    "input-groups.less",
-    "navs.less",
-    "navbar.less",
-    "breadcrumbs.less",
     "pagination.less",
-    "pager.less",
-    "labels.less",
-    "badges.less",
-    "jumbotron.less",
-    "thumbnails.less",
-    "alerts.less",
-    "progress-bars.less",
-    "media.less",
-    "list-group.less",
     "panels.less",
-    "responsive-embed.less",
-    "wells.less",
-    "close.less",
     "component-animations.less",
-    "dropdowns.less",
-    "tooltip.less",
-    "popovers.less",
-    "modals.less",
-    "carousel.less"
+    "popovers.less"
   ],
   "js": [
-    "alert.js",
-    "button.js",
-    "carousel.js",
-    "dropdown.js",
-    "modal.js",
     "tooltip.js",
     "popover.js",
     "tab.js",
-    "affix.js",
-    "collapse.js",
-    "scrollspy.js",
-    "transition.js"
+    "collapse.js"
   ],
-  "customizerUrl": "http://getbootstrap.com/customize/?id=a1f8af4cd5c932d480fede612bbea325"
+  "customizerUrl": "http://getbootstrap.com/customize/?id=a2a9bf8ea73d008dd59a5c6cd45dc9a6"
```

Comment:
There is a difference in the printed version of the html results page.

In `Test ID` we have (on master): 
`1-passtest.py:PassTest.test (../test-results/1-passtest.py:PassTest.test)`

With this commit, `Test ID` becomes:
`1-passtest.py:PassTest.test`

The same happens to the `debug.log` item. That's all I see changing with this commit.